charts/prometheus/templates/cm.yaml

```
@@ -2,9 +2,9 @@
 apiVersion: v1
 kind: ConfigMap
 metadata:
-{{- with .Values.server.configMapAnnotations }}
+{{- with .Values.server }}
   annotations:
-    {{- toYaml .Values.server.configMapAnnotations | nindent 4 }}
+    {{- toYaml .configMapAnnotations | nindent 4 }}
```

... and correct the scope here:
```suggestion
    {{- toYaml . | nindent 4 }}
```